src/BenchmarksDriver/Program.cs

```diff
@@ -87,6 +87,9 @@ public static int Main(string[] args)
                 "Source dependency. Format is 'repo@branchOrCommit'. " +
                 "Repo can be a full URL, or a short name under https://github.com/aspnet.",
                 CommandOptionType.MultipleValue);
+            var threadCountOption = app.Option("-t|--threadCount",
+                "Kestrel ThreadCount.",
+                CommandOptionType.SingleValue);
```

Comment:
Since the option only applies to Kestrel (not HttpSysServer), I'm planning to rename it to `--kestrelThreadCount`.  I will also remove the short version `-t` to avoid confusion.  I'd rather leave the existing `--threads` option for wrk threads alone to minimize churn.